### ISSUE TITLE
Parameters ignored when in workspace mode

### ISSUE THREAD

**Issue by Sytten** [typedoc2md/typedoc-plugin-markdown#697]
### What package is the bug related to?

typedoc-plugin-markdown

### Describe the issue

When you run typedoc with entrypoint `packages`, some options are just ignored like `parametersFormat`, `hidePageHeader`, etc. I am not sure why that is.

### Expected behavior

Ideally all parameters should be respected.

**Comment by tgreyuk** [comment]
@Sytten thanks - this is an area that could be better documented. 

Currently with `packages` entrypoints the options for this plugin need to be set in the root `typedoc.json` and are expected to be global across all packages. Here is a working example - https://github.com/typedoc2md/typedoc-plugin-markdown-examples/blob/main/examples/packages/typedoc.json and example output https://github.com/typedoc2md/typedoc-plugin-markdown-examples/tree/main/examples/packages/docs. 

Basically all options regarding converting the each package are set in individual packages, and TypeDoc reads these options when converting and then destroys them before passing to the renderer which is when this plugin comes into effect.

**Comment by Sytten** [comment]
Make sense do you want me to close the issue?
